sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/Models/AnalyzeOperationResult_internal.cs

```diff
@@ -12,13 +12,31 @@ namespace Azure.AI.FormRecognizer.Models
     /// <summary> Status and result of the queued analyze operation. </summary>
     internal partial class AnalyzeOperationResult_internal
     {
+        /// <summary> Initializes a new instance of AnalyzeOperationResult_internal. </summary>
+        internal AnalyzeOperationResult_internal()
```

Comment:
internal is default for output models and public for input/input-output if parameterless ctor is defined in customization code non would be generated.